source/numerics/vector_tools_boundary.inst.in

```diff
@@ -19,15 +19,15 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS;
 #if deal_II_dimension <= deal_II_space_dimension
     namespace VectorTools \{
       template
-        void interpolate_boundary_values
+        void interpolate_boundary_values<double>
```

Comment:
another example of a non-deducible context. the changes in this file are also all avoidable if you make `number` deducible.